frontend/components/forms/fields/InputField/InputField.jsx

```diff
@@ -196,6 +196,9 @@ class InputField extends Component {
       "labelTooltipPosition",
     ]);
 
+    // FIXME: Why doesn't this pass onBlur and other props down if the type is textarea. Do we want
+    // to change that? What might break if we do?
+
```

Comment:
id imagine we do want to pass the same props but I think its the right call to do this in a different PR so we can test the changes work correctly.